apis/mist/mist.go

```diff
@@ -333,6 +344,26 @@ func (mapi *API) NukeStream(name string) error {
 	return err
 }
 
+func (mapi *API) StartPush(stream string, target string) error {
+	glog.Infof("Starting push for '%s'", stream)
+	reqs := &pushStartReq{
+		Authorize: &authReq{
+			Username: mapi.login,
+			Password: mapi.challengeRepsonse,
+		},
+		PushStart: &pushStartAction{
+			Params: map[string]string{
+				"video": "854x480",
```

Comment:
Looks like values here doesn't matter? Can we put comment about it if so?

Reply:
Ah yep - I thought maybe that was where the video selector goes. But it goes in the target query parameter.